ontologies/Items.ttl

```diff
@@ -0,0 +1,171 @@
+	
+wi:Work rdf:type owl:Class ;
+	rdfs:label "Work"@en ;
+	rdfs:comment "An individual work such as a text, archive item or picture; or a grouping of individual works (so, for instance, an archive collection counts as a work, as do all the series and individual files within it).  Each work may exist in multiple instances (e.g. copies of the same book).  N.B. this is not synonymous with "work" as that is understood in the International Federation of Library Associations and Institutions' Functional Requirements for Bibliographic Records model (FRBR) but represents something lower down the FRBR hierarchy, namely manifestation. Groups of related items are also included as works because they have similar properties to the individual ones."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:Identifier rdf:type owl:Class ;
+	rdfs:label "Identifier"@en ;
+	rdfs:comment "A unique system-generated identifier that governs interaction between systems and is regarded as canonical within the Wellcome data ecosystem."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .		
+
+wi:SourceIdentifier rdf:type owl:Class ;
+	rdfs:label "SourceIdentifier"@en ;
+	rdfs:comment "An identifier used within one of the Wellcome Library's databases or other systems to refer to works or items within the Library's holdings, whether for internal technical purposes or for human citation."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .    
+
+wi:SourceIdentifierAuthority rdf:type owl:Class;
+	rdfs:label "SourceIdentifierAuthority"
+	rdfs:comment "An indication of the origin of SourceIdentifiers: that is, a note of whether this identifier originates in the Sierra bibliographic system, the CALM archive management system, and so forth, and thus an indication of the basic assumptions that lie behind it."@en ;
+	rdfs:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:Classmark rdf:type owl:Class ;
+	rdfs:label "Classmark"@en ;
+	rdfs:comment "A code that relates an item to a classification of library holdings, often but not always relating to a subject-based scheme of classification and arrangement such as Barnard or the National Library of Medicine classification."@en ;
+ 	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+	
+wi:FileFormat rdf:type owl:Class ;
+	rdfs:label "FileFormat"@en ;
+	rdfs:comment "The file format of an item that comprises a digital object."@en ;
+ 	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:Location rdf:type owl:Class ;
+	rdfs:label "Location"@en ;
+	rdfs:comment "The broad division of the Library or its storage areas within which an item is held: for instance, the History of Medicine collection, or the Closed Store Archives and Manuscripts material."@en ;
+ 	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:RequestStatus rdf:type owl:Class ;
```

Comment:
Good point: Missing is a particularly good example of a status that isn’t request.  I’ll do that.